machinist/test/test_fsm.py

```diff
@@ -868,7 +868,6 @@ def test_terminalLogging(self, logger):
                     u"action_type": u"fsm:initialize",
                     u"action_status": u"succeeded",
                     u"task_uuid": logger.messages[0][u"task_uuid"],
-                    u"task_level": u"/",
```

Comment:
Mmmm. Or maybe we could switch to using one of the higher level Eliot testing APIs that will parse the whole action for you.